internal/statestore/backfill.go

```
@@ -59,6 +61,8 @@ func (rb *redisBackend) CreateBackfill(ctx context.Context, backfill *pb.Backfil
 	if res.(int64) == 0 {
 		return status.Errorf(codes.AlreadyExists, "backfill already exists, id: %s", backfill.GetId())
 	}
+
+	acknowledgeBackfill(redisConn, backfill.GetId())
```

you are ignoring error here

---

yes, thanks for noticing, will update

---

TestCreateBackfill should be updated according to this new behavior.